src/assisted_test_infra/test_infra/helper_classes/base_cluster.py

```diff
@@ -153,6 +153,11 @@ def set_hostnames_and_roles(self):
             role = consts.NodeRoles.MASTER if consts.NodeRoles.MASTER in name else consts.NodeRoles.WORKER
             self._infra_env.update_host(host_id=host.get_id(), host_role=role, host_name=name)
 
+    def set_installer_args(self):
+        hosts = self.to_cluster_hosts(self.api_client.get_cluster_hosts(self.id))
+        for host in hosts:
```

Comment:
The way the code written should be :
if not self._config.host_installer_args
   ---> running the loop ...